crates/ruff_python_trivia_integration_tests/tests/simple_tokenizer.rs

```diff
@@ -23,7 +22,7 @@ impl TokenizationTestCase {
     }
 
     fn tokenize_reverse(&self) -> Vec<SimpleToken> {
-        let program = parse_module(self.source).expect("Input to be a valid Python program");
+        let program = parse_unchecked(self.source, Mode::Module);
         BackwardsTokenizer::new(self.source, self.range, program.comment_ranges()).collect()
```

Comment:
We're only interested in the comment ranges and the source can contain syntax errors.